Files/Views/LayoutModes/GenericFileBrowser.xaml

```diff
@@ -251,6 +251,16 @@
                     <FontIcon FontFamily="{StaticResource CustomGlyph}" Glyph="&#xF108;" />
                 </MenuFlyoutItem.Icon>
             </MenuFlyoutItem>
+            <MenuFlyoutItem
+                x:Name="OpenFileLocation"
+                x:Uid="BaseLayoutItemContextFlyoutOpenFileLocation"
```

Comment:
Add ability to translate this text.

Reply:
I added entries for de-DE and en-US in the corresponding Resource files.